src/main/java/com/hotels/beeju/extensions/ThriftHiveMetaStoreJUnitExtension.java

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (C) 2015-2019 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.beeju.extensions;
+
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.hotels.beeju.core.ThriftHiveMetaStoreCore;
+
+/**
+ * A JUnit Extension that creates a Hive Metastore Thrift service backed by a Hive Metastore using an HSQLDB in-memory
```

Comment:
we're not using HSQLDB but Derby: can please change it to `in-memory database` no reason to mention the type. I think I neglected to update this when we moved to Derby, sorry :(